provers/sp1/guest/src/main.rs

```diff
@@ -1,5 +1,5 @@
 #![no_main]
-harness::entrypoint!(main, tests, zk_op::tests);
+sp1_zkvm::entrypoint!(main);
```

Comment:
This is temporary workaround. I need some time to figure out why `harness::entrypoint!(main, tests, zk_op::tests)` is broken.